Content.Shared/Construction/EntitySystems/AnchorableSystem.cs

```diff
@@ -143,10 +143,7 @@ private void OnAnchorComplete(EntityUid uid, AnchorableComponent component, TryA
         var rot = xform.LocalRotation;
         xform.LocalRotation = Math.Round(rot / (Math.PI / 2)) * (Math.PI / 2);
 
-        if (TryComp<PullableComponent>(uid, out var pullable) && pullable.Puller != null)
-        {
-            _pulling.TryStopPull(uid, pullable);
-        }
```

Comment:
TryStopPull will get his PullableComponent no need to it verbose.
pullable.Puller !=null is also done inside  TryStopPull now